models/intermediate/int_iterable__user_event_metrics.sql

```diff
@@ -17,12 +17,18 @@ with user_campaign as (
         max(last_event_at) as last_event_at
 
         {% for col in user_campaign_columns %}
-            {% if col.name|lower not in ['unique_user_key', 'user_id','_fivetran_user_id','user_email', 'user_full_name', 'campaign_id', 'campaign_name', 'recurring_campaign_id', 
-                                        'recurring_campaign_name', 'first_event_at', 'last_event_at', 'template_id', 'template_name',
-                                        'experiment_id'] %}
-        , sum( {{ col.name }} ) as {{ col.name }}
+            {% if col.name|lower not in ['unique_user_key', 'user_id', '_fivetran_user_id', 'user_email', 'user_full_name', 'campaign_id', 'campaign_name', 'recurring_campaign_id', 'recurring_campaign_name', 'first_event_at', 'last_event_at', 'template_id', 'template_name'] %}
+                {% if col.name|lower == 'experiment_id' %}
```

Comment:
Similar note to `int_iterable__campaign_events`.

Reply:
Added experiment_id back to the exclusion